From bcf6972fea5bb6a2452bc9686f13170fd276f452 Mon Sep 17 00:00:00 2001
From: Peter Williams <peter@newton.cx>
Date: Mon, 3 Sep 2018 17:38:10 -0400
Subject: [PATCH 6/9] Fix an inconsistent "noreturn" marking in gettext-tools

Another inconsistency in labeling that MSVC must be pickier about than it used
to be.
---
 gettext-tools/gnulib-lib/xalloc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gettext-tools/gnulib-lib/xalloc.h b/gettext-tools/gnulib-lib/xalloc.h
index f4a329e..a38dcf1 100644
--- a/gettext-tools/gnulib-lib/xalloc.h
+++ b/gettext-tools/gnulib-lib/xalloc.h
@@ -60,7 +60,7 @@ extern "C" {
    in charge of honoring the three previous items.  This is the
    function to call when one wants the program to die because of a
    memory allocation failure.  */
-extern void xalloc_die (void)
+extern _Noreturn void xalloc_die (void)
 #if (__GNUC__ > 2 || (__GNUC__ == 2 && __GNUC_MINOR__ >= 5)) && !__STRICT_ANSI__
      __attribute__ ((__noreturn__))
 #endif
-- 
2.17.1

